docs/security/authzPolicyStorage.md

```diff
@@ -0,0 +1,75 @@
+# Authorization Policy Storage
+
+The authorization policy file is stored in a file called _WEB-INF/classes/security-policy.properties_ stored under the application's WAR structure.
+
+Here is an example of a security policy:
+
+```
+# Role "admin"
+role.admin.home=Home
+role.admin.priority=1
+role.admin.permission.perspective.read=true
+role.admin.permission.perspective.read.Dashboard=false
+
+# Role "user"
+role.user.home=Dashboard
+role.user.priority=0
+role.user.permission.perspective.read=false
+role.user.permission.perspective.read.Home=true
+role.user.permission.perspective.read.Dashboard=true
+```
+
+The format of the key/value pairs is:
+
+`classifier.identifier.setting.extra=value`
+
+Where:
+
+* `classifier` = role|group
+* `identifier` = An existing role or group identifier (depending on the classifier type)
+* `setting` = home|priority|permission
+* `extra` = Extra setting information. Mandatory for instance to define the permission's name
+* `value` = The setting value (depends on the setting selected). Value expected per setting type:
+    * `home`: An existing perspective identifier to redirect after login
+    * `priority`: An integer indicating how priority is this role|group compared to others. Used for conflict resolution.
+    * `permission`: A name representing a specific feature or capability over a given resource.
```

Comment:
This should describe the permission value not the permission i.e. A boolean indicating...